tests/test_blueprint_copy.py

```diff
@@ -74,3 +74,42 @@ def handle_request(request):
     assert "test_bp_copy.test_bp4.handle_request" in route_names
     assert "test_bp_copy.test_bp5.handle_request" in route_names
     assert "test_bp_copy.test_bp6.handle_request" in route_names
+
+
+def test_bp_copy_with_route_overwriting(app: Sanic):
```

Comment:
Can you also add a `test_bp_copy_without_route_overwriting` test that specifcally is set to `False` and catches/asserts on the exception?

Reply:
added